eks-distro-base/Makefile

```
@@ -36,16 +36,12 @@ local-images:
 .PHONY: images
 images:
 	# Sleeping until buildkit daemon is running on the other container
-	sleep 10
-	buildctl \
-		build \
-		--frontend dockerfile.v0 \
-		--opt platform=linux/amd64,linux/arm64 \
-		--opt build-arg:BASE_IMAGE=$(BASE_IMAGE) \
-		--local dockerfile=./ \
-		--local context=. \
-		--output type=image,oci-mediatypes=true,name=$(IMAGE),push=true
-	./update_base_image.sh $(IMAGE_TAG)
+	kubectl build --registry-secret ecr-public \
```

```suggestion
	buildctl build --registry-secret ecr-public \
```
Did we want to vend this image? Are we thinking in terms of how we vend the kube-proxy-base and go-runner base images?

---

Yea what are we trying to change here? I don't think kubectl supports this command so might be a mistake?

---

Sorry again, unrelated change because buildkit locally seems to have problems sometimes doing `yum update` getting mirrors. The kubectl build is an add-on.